### PR TITLE
drivers: Add Home Assistant driver

### PR DESCRIPTION
**Pull request by fooishbar**
Add a very simple driver which allows using the Home Assistant switch API to bridge to e.g. Zigbee devices.

### CODE REVIEW

File: pdudaemon/drivers/homeassistant.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+log = logging.getLogger("pdud.drivers." + os.path.basename(__file__))
+
+
+class HomeAssistantHTTP(PDUDriver):
+    def __init__(self, hostname, settings):
+        """Communicate with custom devices flashed with ESPHome firmware, specifically
+        Switch components (e.g. relays, GPIO).
+        To use this driver, the `web_server` stanza must be defined in the ESPHome
+        configuration.
+        """
```
Comment (jluebbe) [inline pdudaemon/drivers/homeassistant.py:38]: Please update this comment.

**Review by jluebbe — CHANGES_REQUESTED** [review]
Looks good otherwise.

**Comment by fooishbar** [comment]
@jluebbe done now, thanks a lot

**Comment by fooishbar** [comment]
Pushed a simple change to allow using the `hostname` config variable to override this.

**Review by Emantor — APPROVED** [review]

**Review by jluebbe — APPROVED** [review]

**Pull request merged**